### ISSUE TITLE
Comptibility issue during latest upgrade with ReminderFox

### ISSUE THREAD

**Issue by pcahan** [Geek-in-Training/Firefox-Addon-Bar--Restored#32]
When latest upgrade installed, ReminderFox icon/text failed to maintain options in AddOnBar.
Fixable by removing and re-adding and customizing.

**Comment by Geek-in-Training** [comment]
Hmm, strange.

Do you mean latest Firefox update or latest addon update?

**Comment by pcahan** [comment]
Latest addon update
Of course, I have the latest Firefox as well, but I'm not sure that is part of the issue.

**Comment by Geek-in-Training** [comment]
No, it probably wouldn't be Firefox alone.  I mainly asked because I wasn't sure if you had the latest version of the addon, and then Firefox updated, or vice-versa.

I'll see if I can figure out why that would happen. 

Thanks for letting me know!
